depinject/inject.go

```diff
@@ -31,40 +31,38 @@ func inject(loc Location, debugOpt DebugOption, config Config, outputs ...interf
 		return err
 	}
 
+	// always generate graph on exit
+	defer cfg.generateGraph()
```

Comment:
why is this getting called inside `inject`? shouldn't it have already been called elsewhere?

Reply:
Nope, because we add the graph to `c.visualizers` during `cfg.onError.applyConfig(cfg)` when using `OnError`.
There was nothing to be generated when it was called before.